packages/contact-center/cc-components/src/components/task/task.types.ts

```diff
@@ -477,14 +477,34 @@ export type CallControlComponentProps = Pick<
   | 'consultTransferOptions'
 >;
 
+export type OutdialAniEntry = {
```

Comment:
Why do we have this in two places? I saw one in the component itself. Ideally it should be in task types and get imported in the out dial UI component (cc-components)